docs/enriching-your-data/available-enrichments/currency-conversion-enrichment/index.md

```diff
@@ -4,11 +4,22 @@ date: "2020-02-14"
 sidebar_position: 2
 ---
 
-This enrichment uses [Open Exchange Rates](https://openexchangerates.org/) to convert the currencies used in transactions. It requires an Open Exchange Rates.
+This enrichment uses [Open Exchange Rates](https://openexchangerates.org/) to convert the currencies used in transactions. It requires an Open Exchange Rates account and API key.
 
 When transactional data is collected in multiple currencies, it can be useful to convert it in the one that is used for reporting for instance. This could help to lower discrepancies when reporting revenue amounts across multiple currencies.
 
-The conversion is done by the enrich job at processing time with Open Exchange Rates API so that it can be stored directly with the right currency in the database.
+:::caution Limitations
+
+This is an older and less actively maintained enrichment, and as such it has several limitations.
+
+* **It can only use the exchange rates from the end of the day prior to the event’s `collector_tstamp`.** It does not support same-day morning rates or different timestamp fields.
```

Comment:
Isn't same-day morning rate the same thing as end-of-day prior?  I thought markets opened where they finished the previous day?  Forgive my ignorance, I don't know much about exchange rates!

Reply:
I am actually not sure, saw some conflicting info. I’ll change to a different example